docker/docker-entrypoint.sh

```diff
@@ -3,19 +3,13 @@
 set -e
 
 echo "setting up the database"
-django-admin.py migrate
+django-admin.py migrate --noinput
 django-admin.py collectstatic --noinput
 
 echo "from django.contrib.auth.models import User
 if not User.objects.filter(username='admin').count():
     User.objects.create_superuser('admin', 'admin@example.com', 'pass')
 " | django-admin.py shell
 
-echo "=> Starting nginx"
-nginx
-
 echo "=> Starting Supervisord"
-supervisord -c /etc/supervisord.conf
-
-echo "=> Tailing logs"
-tail -qF /var/log/supervisor/*.log
+exec supervisord -c /etc/supervisor/supervisord.conf
```

Comment:
`exec` replaces the bash process with the process being `exec`ed so you only have the one process (supd in this case) instead of `bash` with a child. This is important for signal propagation and such.